ocfweb/main/templates/main/home.html

```diff
@@ -7,11 +7,24 @@
 {% block hero %}
 <div class="ocf-penguin-swing"></div>
 
-<div class="ocf-hero">
+<div class="ocf-hero" style="background-color:#888">
     <div class="container">
         <div class="row">
             <div class="col-sm-8">
-                <h1>Welcome to the Open Computing Facility!</h1>
+                <h1>The OCF Stands with Black Lives Matter</h1>
+                <p>We at the OCF believe in the idea of freedom. Both free technology and free people, regardless of color, gender, or sexuality. Here are some links to some resources.</p>
```

Comment:
either some links to resources or links to some resources {for}.

Reply:
yeah I initially changed the scss file, but I realized it changed the color which is used in other places. My impression was to just change it for the home page where the relevant info is, so I tried to not change the scss file. But if we wanna do a dark theme and just change the whole thing, I can do that.